apps/pie-docs/src/assets/js/nav-toggle-listener.js

```diff
@@ -0,0 +1,14 @@
+const navEl = document.querySelector('#navToggle');
```

Comment:
Is this JS just to add JS for the tests to hook onto?

Depending on how the selector engine in wdio works, there's probably a way of just checking the `c-navToggle-input:checked` as that should be either selected or not to show if the menu is open or closed. Would save having to add a JS file for the tests that way.

Reply:
Good idea, didn't think of this!